ipinfo/core.go

```diff
@@ -6,25 +6,27 @@ import (
 
 // Core represents data from the Core API.
 type Core struct {
-	IP          net.IP       `json:"ip" csv:"ip"`
-	Hostname    string       `json:"hostname,omitempty" csv:"hostname"`
-	Bogon       bool         `json:"bogon,omitempty" csv:"bogon"`
-	Anycast     bool         `json:"anycast,omitempty" csv:"anycast"`
-	City        string       `json:"city,omitempty" csv:"city"`
-	Region      string       `json:"region,omitempty" csv:"region"`
-	Country     string       `json:"country,omitempty" csv:"country"`
-	CountryName string       `json:"country_name,omitempty" csv:"country_name"`
-	IsEU        bool         `json:"isEU,omitempty" csv:"isEU"`
-	Location    string       `json:"loc,omitempty" csv:"loc"`
-	Org         string       `json:"org,omitempty" csv:"org"`
-	Postal      string       `json:"postal,omitempty" csv:"postal"`
-	Timezone    string       `json:"timezone,omitempty" csv:"timezone"`
-	ASN         *CoreASN     `json:"asn,omitempty" csv:"asn_,inline"`
-	Company     *CoreCompany `json:"company,omitempty" csv:"company_,inline"`
-	Carrier     *CoreCarrier `json:"carrier,omitempty" csv:"carrier_,inline"`
-	Privacy     *CorePrivacy `json:"privacy,omitempty" csv:"privacy_,inline"`
-	Abuse       *CoreAbuse   `json:"abuse,omitempty" csv:"abuse_,inline"`
-	Domains     *CoreDomains `json:"domains,omitempty" csv:"domains_,inline"`
+	IP           net.IP       `json:"ip" csv:"ip"`
+	Hostname     string       `json:"hostname,omitempty" csv:"hostname"`
+	Bogon        bool         `json:"bogon,omitempty" csv:"bogon"`
+	Anycast      bool         `json:"anycast,omitempty" csv:"anycast"`
+	City         string       `json:"city,omitempty" csv:"city"`
+	Region       string       `json:"region,omitempty" csv:"region"`
+	Country      string       `json:"country,omitempty" csv:"country"`
+	CountryName  string       `json:"country_name,omitempty" csv:"country_name"`
+	EmojiFlag    string       `json:"emoji_flag,omitempty" csv:"emoji_flag"`
+	EmojiUnicode string       `json:"emoji_unicode,omitempty" csv:"emoji_unicode"`
```

Comment:
We should put the `CountryFlag` struct here too, replacing these above 2 fields as they'll already be inside of it.